cumulus/parachains/runtimes/coretime/coretime-rococo/src/xcm_config.rs

```diff
@@ -140,15 +140,22 @@ impl Contains<RuntimeCall> for SafeCallFilter {
 
 		matches!(
 			call,
-			RuntimeCall::PolkadotXcm(pallet_xcm::Call::force_xcm_version { .. }) |
-				RuntimeCall::System(
-					frame_system::Call::set_heap_pages { .. } |
+			RuntimeCall::PolkadotXcm(
+				pallet_xcm::Call::force_xcm_version { .. } |
+					pallet_xcm::Call::force_default_xcm_version { .. }
+			) | RuntimeCall::System(
+				frame_system::Call::set_heap_pages { .. } |
 						frame_system::Call::set_code { .. } |
 						frame_system::Call::set_code_without_checks { .. } |
-						frame_system::Call::kill_prefix { .. },
-				) | RuntimeCall::ParachainSystem(..) |
+						frame_system::Call::authorize_upgrade { .. } |
+						frame_system::Call::authorize_upgrade_without_checks { .. } |
```

Comment:
Should we be marking this one with a warning as well? Likewise in the other runtimes.

Reply:
IMO no, the function itself has a warning. It does meet the criteria of being "safe" for the `SafeCallFilter`.